apps/researcher/src/app/[locale]/communities/layout.tsx

```diff
@@ -5,8 +5,8 @@ import {ReactNode} from 'react';
 
 export default function Layout({children}: {children: ReactNode}) {
   return (
-    <body className="bg-consortiumGreen-300 text-consortiumBlue-800">
-      <BaseLayout wrapperClassName="mx-auto h-full min-h-screen flex flex-col justify-stretch items-stretch gap-8 pb-40">
```

Comment:
With this removed, the 'all communities' button on a community page is to high

![image](https://github.com/colonial-heritage/colonial-collections/assets/1481602/9f62dd96-e804-413d-a455-28d7806e9771)